### PR TITLE
(TAU) Graph widget documentation

### PR DESCRIPTION
**Pull request by TomaszLukawskiSam**
### Change Description ###
New documentation page for Graph widget.

- added widget description
- options reference
- method description
- examples

Dedicated for profiles: mobile, wearable

Issue: https://github.com/Samsung/TAU/issues/454

Signed-off-by: Tomasz Lukawski <t.lukawski@samsung.com>

### Bugs Fixed ###
does not refer to a bug

### API Changes ###
does not refer to an ACR

### CODE REVIEW

**Comment by mijin85cho** [comment]
It looks like new pages are added. So, please don't forget to update api/5.5/index.htm also.

File: docs/application/web/api/5.5/ui_fw_api/Mobile_UIComponents/mobile_Graph.htm [hunk 1]
```diff
@@ -0,0 +1,307 @@
...
+			<td class="option"><span style="font-family: Courier New,Courier,monospace">data-value</span></td>
+			<td class="option">string | null</td>
+			<td class="option">null</td>
+			<td class="option">Initial values of Graph widget</td>
+		</tr>
+		<tr>
+			<td class="option"><span style="font-family: Courier New,Courier,monospace">data-mode</span></td>
+			<td class="option">string | null</td>
+			<td class="option">&quot;intermittent&quot;</td>
+			<td class="option">This option determine how the new values will replaced values of Graph widget.<br>
```
Comment (lslachciak) [inline docs/application/web/api/5.5/ui_fw_api/Mobile_UIComponents/mobile_Graph.htm]: determine --> determines

File: docs/application/web/api/5.5/ui_fw_api/Mobile_UIComponents/mobile_Graph.htm [hunk 2]
```diff
@@ -0,0 +1,307 @@
...
+				This option allows also value &quot;continuous&quot; which determine that new values will be appended to Graph</td>
+		</tr>
+		<tr>
+			<td class="option"><span style="font-family: Courier New,Courier,monospace">data-color</span></td>
+			<td class="option">string | null</td>
+			<td class="option">&quot;#0097D8&quot;</td>
+			<td class="option">This option determine color of line or bar which represents data</td>
```
Comment (lslachciak) [inline docs/application/web/api/5.5/ui_fw_api/Mobile_UIComponents/mobile_Graph.htm]: deter,mine --> determines

File: docs/application/web/api/5.5/ui_fw_api/Mobile_UIComponents/mobile_Graph.htm [hunk 3]
```diff
@@ -0,0 +1,307 @@
...
+			<td class="option"><span style="font-family: Courier New,Courier,monospace">data-yinit</span></td>
+			<td class="option">string | null</td>
+			<td class="option">&quot;0&quot;</td>
+			<td class="option">Initial value of Graph on Y axis</td>
+		</tr>
+		<tr>
+			<td class="option"><span style="font-family: Courier New,Courier,monospace">data-axis-x-type</span></td>
+			<td class="option">string | null</td>
+			<td class="option">&quot;time&quot;</td>
+			<td class="option">Define type of values on X axis. Supported values: &quot;time&quot;, &quot;index&quot;</td>
```
Comment (lslachciak) [inline docs/application/web/api/5.5/ui_fw_api/Mobile_UIComponents/mobile_Graph.htm]: define --> defines

File: docs/application/web/api/5.5/ui_fw_api/Wearable_UIComponents/wearable_graph.htm [hunk 4]
```diff
@@ -0,0 +1,306 @@
...
+						<th>Description</th>
+					</tr>
+
+
+					<tr>
+						<td><span style="font-family: Courier New,Courier,monospace">value</span></td>
+						<td>number | array | null</td>
+						<td>Optional</td>
+						<td>null</td>
+						<td>Widget accept data series as array, eg. [1,2,4,5,8]<br>
```
Comment (lslachciak) [inline docs/application/web/api/5.5/ui_fw_api/Wearable_UIComponents/wearable_graph.htm]: accept --> accepts

File: docs/application/web/api/5.5/ui_fw_api/Wearable_UIComponents/wearable_graph.htm [hunk 5]
```diff
@@ -0,0 +1,306 @@
...
+			<td class="option"><span style="font-family: Courier New,Courier,monospace">data-yinit</span></td>
+			<td class="option">string | null</td>
+			<td class="option">&quot;0&quot;</td>
+			<td class="option">Initial value of Graph on Y axis</td>
+		</tr>
+		<tr>
+			<td class="option"><span style="font-family: Courier New,Courier,monospace">data-axis-x-type</span></td>
+			<td class="option">string | null</td>
+			<td class="option">&quot;time&quot;</td>
+			<td class="option">Define type of values on X axis. Supported values: &quot;time&quot;, &quot;index&quot;</td>
```
Comment (lslachciak) [inline docs/application/web/api/5.5/ui_fw_api/Wearable_UIComponents/wearable_graph.htm]: Define->Defines

File: docs/application/web/api/5.5/ui_fw_api/Wearable_UIComponents/wearable_graph.htm [hunk 6]
```diff
@@ -0,0 +1,306 @@
...
+			<td class="option">string | null</td>
+			<td class="option">&quot;intermittent&quot;</td>
+			<td class="option">This option determine how the new values will replaced values of Graph widget.<br>
+				This option allows also value &quot;continuous&quot; which determine that new values will be appended to Graph</td>
+		</tr>
+		<tr>
+			<td class="option"><span style="font-family: Courier New,Courier,monospace">data-color</span></td>
+			<td class="option">string | null</td>
+			<td class="option">&quot;#0097D8&quot;</td>
+			<td class="option">This option determine color of line or bar which represents data</td>
```
Comment (lslachciak) [inline docs/application/web/api/5.5/ui_fw_api/Wearable_UIComponents/wearable_graph.htm]: detemirne --> determines

File: docs/application/web/api/5.5/ui_fw_api/Mobile_UIComponents/mobile_Graph.htm [hunk 7]
```diff
@@ -0,0 +1,307 @@
...
+		<li><a href="#options-for widget-graph-mobile">Options</a></li>
+
+		<li><a href="#methods-graph-mobile">Methods</a></li>
+
+</ol>
+
+<h2><a id="default-selectors-graph-mobile"></a>Default Selectors</h2>
+
+<p>Elements with the <code>class=&quot;ui-graph&quot;</code> attribute is building as Graph widget.<br>
+Changing the value of <code>data-graph=&quot;line&quot;</code> applies a variety of graph style.<br>
```
Comment (lslachciak) [inline docs/application/web/api/5.5/ui_fw_api/Mobile_UIComponents/mobile_Graph.htm]: style --> styles

**Comment by lslachciak** [comment]
staging urls:
https://docs.stg.tizen.org/staging/897/application/web/api/5.5/ui_fw_api/Mobile_UIComponents/mobile_Graph.htm
https://docs.stg.tizen.org/staging/897/application/web/api/5.5/ui_fw_api/Wearable_UIComponents/wearable_graph.htm

File: docs/application/web/api/5.5/ui_fw_api/Mobile_UIComponents/mobile_Graph.htm [hunk 8]
```diff
@@ -0,0 +1,307 @@
...
+
+		<li><a href="#options-for widget-graph-mobile">Options</a></li>
+
+		<li><a href="#methods-graph-mobile">Methods</a></li>
+
+</ol>
+
+<h2><a id="default-selectors-graph-mobile"></a>Default Selectors</h2>
+
+<p>Elements with the <code>class=&quot;ui-graph&quot;</code> attribute is building as Graph widget.<br>
```
Comment (lslachciak) [inline docs/application/web/api/5.5/ui_fw_api/Mobile_UIComponents/mobile_Graph.htm]: is building ..
->
are build...

File: docs/application/web/api/5.5/ui_fw_api/Mobile_UIComponents/mobile_Graph.htm (hunk 1 above)
Comment (TomaszLukawskiSam) [reply docs/application/web/api/5.5/ui_fw_api/Mobile_UIComponents/mobile_Graph.htm]: Done

File: docs/application/web/api/5.5/ui_fw_api/Mobile_UIComponents/mobile_Graph.htm (hunk 2 above)
Comment (TomaszLukawskiSam) [reply docs/application/web/api/5.5/ui_fw_api/Mobile_UIComponents/mobile_Graph.htm]: Done

File: docs/application/web/api/5.5/ui_fw_api/Wearable_UIComponents/wearable_graph.htm (hunk 4 above)
Comment (TomaszLukawskiSam) [reply docs/application/web/api/5.5/ui_fw_api/Wearable_UIComponents/wearable_graph.htm]: Done

Comment (TomaszLukawskiSam) [reply docs/application/web/api/5.5/ui_fw_api/Wearable_UIComponents/wearable_graph.htm]: Done

File: docs/application/web/api/5.5/ui_fw_api/Wearable_UIComponents/wearable_graph.htm (hunk 5 above)
Comment (TomaszLukawskiSam) [reply docs/application/web/api/5.5/ui_fw_api/Wearable_UIComponents/wearable_graph.htm]: Done, Done

File: docs/application/web/api/5.5/ui_fw_api/Wearable_UIComponents/wearable_graph.htm (hunk 6 above)
Comment (TomaszLukawskiSam) [reply docs/application/web/api/5.5/ui_fw_api/Wearable_UIComponents/wearable_graph.htm]: Done

File: docs/application/web/api/5.5/ui_fw_api/Mobile_UIComponents/mobile_Graph.htm (hunk 7 above)
Comment (TomaszLukawskiSam) [reply docs/application/web/api/5.5/ui_fw_api/Mobile_UIComponents/mobile_Graph.htm]: Done. Done

File: docs/application/web/api/5.5/ui_fw_api/Mobile_UIComponents/mobile_Graph.htm (hunk 8 above)
Comment (TomaszLukawskiSam) [reply docs/application/web/api/5.5/ui_fw_api/Mobile_UIComponents/mobile_Graph.htm]: Done

File: docs/application/web/api/5.5/ui_fw_api/Mobile_UIComponents/mobile_Graph.htm (hunk 3 above)
Comment (TomaszLukawskiSam) [reply docs/application/web/api/5.5/ui_fw_api/Mobile_UIComponents/mobile_Graph.htm]: Done, Done

**Comment by lslachciak** [comment]
Looks good

**Comment by mijin85cho** [comment]
Don't you need to update `wearable_extended_component_list.htm` and 
`mobile_extended_component_list.htm` also?

**Comment by lslachciak** [comment]
@mijin85cho thaqnk you for remark, you are right.
We should update also this two files.

**Comment by mijin85cho** [comment]
Thanks for the update.

**Pull request merged**